chimingxuan/client/src/client.cpp

```diff
@@ -0,0 +1,79 @@
+#include "ros/ros.h"
+#include "std_msgs/String.h"
+#include "server/Register.h"
+#include "server/Unregister.h"
+
+#include <signal.h>
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Found C system header after other header. Should be: client.h, c system, c++ system, other.  [build/include_order] [4]